docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# Contributing Guidelines
+
+*We're so excited that you're interested in contributing to Neurobagel!* :partying_face:
+
+We appreciate all contributions, and hope that the below guidelines will make it as easy as possible for you to contribute to the [Neurobagel codebase](https://github.com/neurobagel) and to ensure your contribution can be easily integrated.
+
+## Identifying an issue to work on
+
+The easiest way to get started contributing is to explore the list of open issues in one of our [GitHub repositories](https://github.com/orgs/neurobagel/repositories). 
+For example, see the [open issues for the Neurobagel API](https://github.com/neurobagel/api/issues).
```

Comment:
Maybe instead: look for "good first issue" labels?

Reply:
agree. I mainly meant: keeping this generic across repos seems clearer to me. But :cherries: